docs/backend-services/graphql/GraphQL-Filtering.md

```diff
@@ -131,4 +132,27 @@ this.gridOptions = {
     }
   }
 }
+```
+
+### Override the filter query
+
+Column filters may have a `Custom` Operator, that acts as a placeholder for you to define your own logic. To do so, the easiest way is to provide the `filterQueryOverride` callback in the GraphQL Options. This method will be called with `GraphqlFilterQueryOverrideArgs` to let you decide dynamically on how the filter should be assembled.
+
+E.g. you could listen for a specific column and the active `OperatorType.custom` in order to switch the filter to an SQL LIKE search in GraphQL:
+
+> **Note** technically speaking GraphQL isn't a database query language like SQL, it's an application query language.
+> What that means is that GraphQL won't let you write arbitrary queries out of the box. It will only support the types of queries defined in your GraphQL schema.
+> see this SO: https://stackoverflow.com/a/37981802/1212166
```

Comment:
Depending on your GraphQL backend, chances are high a regex filter is supported. e.g here in Hasura https://hasura.io/docs/latest/queries/postgres/filters/text-search-operators/#_regex

so with that said, you could pass on the constructed Regex, like I'm doing in the OData sample, and have that perform the same feature as SQL LIKE would do. Keep in mind you're most likely looking for an case insenstive variante by either attaching `/i` or having an operator called `IRegex`  provided by your backend.

Reply:
@zewa666 I wasn't aware of that, so what are you suggesting to change then? Should I reference your link as well or replace my link? I'd be happy to just copy+paste what you have in mind 😉